transports/rabbitmq/index_quorumqueues_rabbit_[7,).partial.md

```diff
@@ -0,0 +1,11 @@
+### Quorum queues
```

Comment:
might be better to move this to https://docs.particular.net/transports/rabbitmq/cluster ?